common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -3,6 +3,7 @@
 #include <cstdlib>
 #include <cstring>
 #include <dlfcn.h>
+#include <kokkos_core.hpp>
```

Comment:
That header file does not exist and even if you spelled it properly that is not the right fix.